src/ios/view.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * test view
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { UIScreen } from 'UIKit/UIScreen';
+import { UIColor } from 'UIKit/UIColor';
+import { UIView } from 'UIKit/UIView';
+
+export default class View {
```

Comment:
Sorry, my comment is confusing.

What I've been doing is if a module exports a single class, then use `default` unless you intend to have your module be consumed by a CommonJS loader (i.e. Node's `require()`).

Your imports at the top pull in `UIScreen` which means `UIScreen` is not `UIKit/UIScreen`'s default export. If `UIKit/UIScreen` only exports a single class, then it should probably be the default export and you should not use `{ }` around `UIScreen`. The point is be consistent internally. Adjust your public API as how it's going to be consumed.

Think of all ES6 modules exporting a single object that has a `default` key with the default exported thing. There can only be one default export. All other exports are just key/values of the exported object.

```javascript
export default class View {}

export class UIScreen {}
```

```javascript
import View, { UIScreen } from './view';
// or
var View = require('./view').default;
var UIScreen = require('./view').UIScreen;
```

Exporting a default object when being consumed by Node's `require()` makes things hard to read. You don't want to put a bunch of `.default`s everywhere. It's better to be verbose. If your module is going to be consumed by another ES6 module, then just stick with a default export.

So, I would leave `view.js` alone and consider making `UIScreen`, `UIColor`, and `UIView` export a default value.